changes/1013.change.md

```diff
@@ -0,0 +1 @@
+Removed some redundant/unused fields from CommandBuilder.
```

Comment:
yeah idk what category to put this under cause its not really breaking (CommandBuilder was already broken for ppl who were directly using its not useful by itself so ppl will have moved away from using it directly as its a base class) and its not really a feature either

Reply:
Can be marked as a bugfix then